src/scss/_learn.scss

```diff
@@ -0,0 +1,4 @@
+.learn-card {
+    box-shadow: 0px 6px 36px rgba(0, 0, 0, 0.1);
```

Comment:
do these need to be different than whatever our standard card styles are?

Reply:
These are the values from the marketing cards referenced in https://github.com/pulumi/pulumi-hugo/pull/552#issuecomment-933627048. I didn't see a specific style to add off the cuff, so I grabbed the `box-shadow` values for now to add to the card to match the ideal shadow.

Reply:
Right -- these cards should probably be using either our already-existing `.card` style (which gives you a "marketing style" card) or an `@apply shadow-xl` or some similar Tailwind-ism (which we use ~everywhere else).

Reply:
Ah, gotcha. I was seeing some oddities when I used the `.card` style, but it might have been an artifact on my system while I was cleaning up things. I'll give it a shot. The TailwindCSS shadow was the one that it had.

Reply:
Sounds good, thanks!